src/utils/types.ts

```diff
@@ -35,3 +35,11 @@ export interface DataDictionary {
     };
   };
 }
+
+export enum View {
+  Landing = 'landing',
+  Upload = 'upload',
+  ColumnAnnotation = 'columnAnnotation',
+  ValueAnnotation = 'valueAnnotation',
+  Download = 'download',
+}
```

Comment:
not sure `types.ts` is the best place for that. `constants.ts` maybe?

Reply:
Isn't an enum a kind of type?